data.sql

```diff
@@ -5,4 +5,20 @@ VALUES ('Gabumon', DATE '2018-11-15',2,true,8.00);
 INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
 VALUES ('Pikachu', DATE '2021-01-07',1,false,15.04);
 INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
-VALUES ('Devimon', DATE '2017-05-12',5,true,11.00);
\ No newline at end of file
+VALUES ('Devimon', DATE '2017-05-12',5,true,11.00);
+
+/*Add new animal*/
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Charmander', DATE '2020-02-08',0,false,11.00);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Plantom', DATE '2021-11-15',2,true,5.70);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Squirtle', DATE '1993-04-02',3,false,12.13);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
+VALUES ('Angemon', DATE '2005-06-12',1,true,45.00);
+INSERT INTO animal (animal_name,date_of_birth,escape_attempts,neutered,weigth_kg)
```

Comment:
- [ ] I have remarked that some of your values are not negative {`,11.00, 5.7, 12.13, 45.0`}, take a look back at the project requirement and update your table data values, please